README.md

```diff
@@ -58,49 +58,41 @@ Usage
 Example setup
 -------------
 
-To build manylinux, macOS, and Windows wheels on Travis CI and upload them to PyPI whenever you tag a version, you could use this `.travis.yml`:
+To build manylinux, macOS, and Windows wheels on Github Actions, you could use this `.github/workflows/wheels.yml`:
 
 ```yaml
-language: python
+name: Build
+
+on: [push, pull_request]
 
 jobs:
-  include:
-    # perform a linux build
-    - services: docker
-    # and a mac build
-    - os: osx
-      language: shell
-    # and a windows build
-    - os: windows
-      language: shell
-      before_install:
-        - choco install python --version 3.8.0
-        - export PATH="/c/Python38:/c/Python38/Scripts:$PATH"
-        # make sure it's on PATH as 'python3'
-        - ln -s /c/Python38/python.exe /c/Python38/python3.exe
-
-env:
-  global:
-    - TWINE_USERNAME=__token__
-    # Note: TWINE_PASSWORD is set to a PyPI API token in Travis settings
-
-install:
-  - python3 -m pip install cibuildwheel==1.7.1
-
-script:
-  # build the wheels, put them into './wheelhouse'
-  - python3 -m cibuildwheel --output-dir wheelhouse
-
-after_success:
-  # if the release was tagged, upload them to PyPI
-  - |
-    if [[ $TRAVIS_TAG ]]; then
-      python3 -m pip install twine
-      python3 -m twine upload wheelhouse/*.whl
-    fi
+  build_wheels:
+    name: Build wheels on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-20.04, windows-2019, macOS-10.15]
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-python@v2
+        name: Install Python
+
+      - name: Install cibuildwheel
+        run: python -m pip install cibuildwheel==1.7.1
+
+      - name: Build wheels
+        env:
+          CIBW_SKIP: "?p25-*" # Skip building Python 2.7 wheels on all platforms
```

Comment:
There's a typo here, it should be 27, not 25. But also I don't think the setting should be written in the config like this, as users might assume that it's required for this setup to work. Also skipping 2.7 would also mandate the removal of the choco install in windows. If the line is there but commented, users can assume it's educational, not required.

Reply:
Oops. How about:

```suggestion
          CIBW_SKIP: "?p27-*" # Skipping Python 2.7 for this example. See docs and examples for details on Python 2.7.
```

This has several benefits:

1. It avoids having to show choco install, which is very verbose for a simple readme example and is wrong for a _large_ portion of Python 2.7 users, since it doesn't support anything above C++03 (so no pybind11, etc).
2. cibuildhwheel will need to support Python 2 as long as the  manylinux wheels have it, but we don't have to include it in the "simplest possible" example in the readme. It's already been tripping up users like in #468 
3. It can remain in the bigger example in the examples folder.

I'd hope most users are past 2.7 support, so this is the new "default", with <50% now needing the Python 2 part, making an example that skips it more useful for a minimal first example. I'll restore it if you want it back, though.

Reply:
Do we really want to do this? I thought the idea was that for this example to be as minimal/general as possible?

Reply:
As I see our general example does not show how to use config variables. Also, we got bug reports like #468. I`m ok with disabling python 2.7 in the initial configuration from readme and does not do this in the example yaml file.

Reply:
Dropping 2.7 via a variable is more minimal than adding a whole step that adds Python 2.7 but only works for C/C++<11. And does removing 2.7 from the readme example really make it less general at the end of 2020? If anything it still shows that we support it (since we remove it via a variable), and highlights that there are caveats to using it (the extra step or steps for C++11+ support).

Reply:
> As I see our general example does not show how to use config variables.

There's a whole section "Options", right underneath the minimal example, so it should be reasonably clear if users read the docs?

> adding a whole step that adds Python 2.7 but only works for C/C++<11

Is that still an issue on GitHub Actions (compared to Travis)? I thought everything was much more streamlined on GHA?
But yes, if that's the case, I can probably agree.

Reply:
> Is that still an issue on GitHub Actions (compared to Travis)?

Oh, I see, the installation step of the ancient Python 2.7 compiler is still there.

Reply:
> There's a whole section "Options", right underneath the minimal example, so it should be reasonably clear if users read the docs?

There is rather a problem that you need to found how to set an environment variable in yaml file. 
I think that it may be unclear that cibuildwheel build wheel on all python version if the environment variable is not set. I see examples with hole os times python version matrix.